install/uninstalling_cluster.adoc

```diff
@@ -31,6 +31,31 @@ installed by Ansible, including:
 The playbook deletes content for any hosts defined in the inventory file
 that you specify when running the playbook.
 
+[IMPORTANT]
+====
+Before you uninstall your cluster, make sure that uninstalling is the best
```

Comment:
It might be implied, because this sentence immediately precedes a list, but perhaps it's better to say something like the following:

> Before you uninstall your cluster, examine the following items to ensure an uninstall is necessary.